ci/pipeline.yml

```diff
@@ -0,0 +1,151 @@
+---
+jobs:
+- name: push-cg-quotas-db-staging
+  plan:
+  - get: cg-quotas-db-staging
+    trigger: true
+  - task: run-tests
+    file: cg-quotas-db-staging/ci/run-tests.yml
+    input_mapping:
+      cg-quotas-db: cg-quotas-db-staging
+  - task: create-db
+    file: cg-quotas-db-staging/ci/create-db.yml
+    input_mapping:
+      cg-quotas-db: cg-quotas-db-staging
+    params:
+      SERVICE_NAME: rds-cg-quotas-db
+      SERVICE_PLAN: shared-psql
+      CF_API_URL: {{staging-cf-api-url}}
+      CF_DEPLOY_USERNAME: {{staging-cf-deploy-username}}
+      CF_DEPLOY_PASSWORD: {{staging-cf-deploy-password}}
+      CF_ORGANIZATION: {{staging-cf-organization}}
+      CF_SPACE: {{staging-cf-space}}
```

Comment:
Not a blocker, but this seems general-purpose enough that we may want to include it in https://github.com/18F/cg-pipeline-tasks.